### PR TITLE
Add support for autotools

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Related to #2

### CODE REVIEW

File: src/terminal.ts [hunk 1]
```diff
@@ -196,6 +196,59 @@ export class FlatpakManifest {
...
+        ['build', ...buildArgs, this.repoDir, 'make', '-p', '-n', '-s'],
+        path.join(this.workspace),
+        this.isSandboxed
+      )
+    )
+
+    commands.push(
+      new Command(
+        'flatpak',
+        ['build', ...buildArgs, this.repoDir, 'make', 'V=0', '-j5', 'install'],
```
Comment (bilelmoussaoui) [inline src/terminal.ts:244]: Do we we want to hardcode the number of builders to 5?

Comment (Hofer-Julian) [reply src/terminal.ts:244]: Good point.
Honestly not sure why GNOME builder chooses five on my machine (pretty confident that no machine has an uneven number of cores).

Can we get the number of cores without an additional dependency?

Comment (Hofer-Julian) [reply src/terminal.ts:244]: I just wonder if we care enough about one aspect of autotools that we want to add logic for that.

Comment (bilelmoussaoui) [reply src/terminal.ts:244]: >  Can we get the number of cores without an additional dependency?

Nodejs has definitely an API for this

>  I just wonder if we care enough about one aspect of autotools that we want to add logic for that.

Well, we will definitely have to do more integration if we want things to work as expected ^^

Comment (Hofer-Julian) [reply src/terminal.ts]: Added https://nodejs.org/api/os.html#os_os_cpus

**Pull request merged**